ministryofjustice/analytics-platform-control-panel-frontend
# homepage for users and superusers

## What

* Revert to having one template for the homepage for users and superusers
* Show superuser functions (list all users, etc) to superusers only
* Show user's apps and buckets on homepage for all users

## How to review

1. check out, run, log in as superuser
2. see your homepage including "Superuser functions" at the top
3. if you can log in as a non superuser, do so and you will see the same homepage without the superuser functions

superuser homepage
---
![screen shot 2017-12-06 at 14 08 48](https://user-images.githubusercontent.com/988436/33665499-0cd8273c-da8f-11e7-9f1c-75deaed5ac44.png)

regular user homepage
---
![screen shot 2017-12-06 at 14 09 04](https://user-images.githubusercontent.com/988436/33665506-110b06bc-da8f-11e7-9235-e773ec02f609.png)
